proto/celestia/qgb/v1/query.proto

```diff
@@ -30,6 +30,11 @@ service Query {
       returns (QueryLatestAttestationNonceResponse) {
     option (google.api.http).get = "/qgb/v1/attestations/nonce/latest";
   }
+  // LatestAttestationNonce queries latest attestation nonce.
+  rpc EarliestAttestationNonce(QueryEarliestAttestationNonceRequest)
+      returns (QueryEarliestAttestationNonceResponse) {
+    option (google.api.http).get = "/qgb/v1/attestations/nonce/earliest";
```

Comment:
[question] did the rename PR not include a rename to the endpoint?

I would expect `/blockstream/v1` and not `/qgb/v1/

Reply:
that would be breaking, we wanted to keep that until v2

Reply:
to clarify the smallest bit, we want to add the consensus breaking things, but we first need the infra to be able to test the upgrade/migrations

Also, if we don't already have an issue for that, can we get one?

Reply:
https://github.com/celestiaorg/celestia-app/issues/2733